packages/yswaps/scripts/execute-fantom-pending-trades.ts

```diff
@@ -93,7 +94,30 @@ async function main() {
           params: [snapshotId],
         });
 
-        // await fantomProvider.sendTransaction(executeTx.data!);
+        const gasParams = {
+          gasPrice: utils.parseUnits(`${gasprice.get()}`, 'gwei'),
+          gasLimit: 1_000_000,
+        };
+
+        const signedTx = await ymech.signTransaction({
+          ...gasParams,
+          to: executeTx.to!,
+          data: executeTx.data!,
+          chainId: 250,
+          nonce: nonce,
+        });
+
+        const tx = await fantomProvider.sendTransaction(signedTx);
+
+        console.log(`[Execution] Transaction set, check at https://ftmscan.com/tx/${tx.hash}`);
+
+        try {
+          await tx.wait(1);
```

Comment:
I see you are not scared of reorgs

Reply:
since FTM is so fast i'd suggest aiming for 20 confirmations
```suggestion
          await tx.wait(1);
```

Reply:
Yeah ! Honestly, it was more to check "it didnt revert" than to check "hey it has been confirmed" ! But 20 confirmations look good !